### ISSUE TITLE
Gallery Widget in Customizer: Errors when trying to insert gallery via customizer

### ISSUE THREAD

**Issue by samhotchkiss** [Automattic/jetpack#1970]
https://make.wordpress.org/flow/2015/04/15/adding-a-widget-via-the-customizer-ipad-air-landscape/#jp-carousel-1391

**Comment by designsimply** [comment]
This is not just broken on mobile. I tested on both iPad and desktop and found that I couldn't see the same notices from ryanboren's screenshot, but I also couldn't get the gallery widget to work.

My testing steps:

1. Go to Jetpack in the dashboard and make sure Tiled Galleries are enabled
1. Go to Plugins and note versions (JP 3.4.1 on WP 4.2-RC1
1. Go to Appearance > Customize > Widgets > Widget area
1. Click the "Add a Widget" button
1. Search for "gallery" and click to add it
1. Click the "Choose Images" button (re-open if needed)
1. Click "Add to Gallery" on the left
1. Select a few images
1. Click the "Add to gallery" button
1. Click the "Insert gallery"
1. Check to see if the widget appears in the widget panel on the left or the live preview on the right
1. Click the "Save & Publish" button
1. Go to the blog home page and try to find the gallery widget
* Result: the gallery widget appears as though it can be added but it never displays in the widget area, the live preview, or the front end after saving

iPad test: https://cloudup.com/cCHmWIKKH0I (2m50s)
Desktop test: https://cloudup.com/cV-lofie3xf (1m43s)

Tested with Safari iOS 8.1.3 on iPad 3rd Gen (Retina) and Firefox 37.0.1 on Mac OSX 10.10.3 with Jetpack 3.4.1 on WP 4.2-RC1 (WP_DEBUG=true).

**Comment by dereksmart** [comment]
Hi @designsimply, 
There is definitely some funkyness going on.  When I tested, I noticed that the gallery images don't show up or save unless you input a Title after adding the images.  Check out my screencast (but ignore the last half where I'm trying to figure out how to stop it :))

https://cloudup.com/caywHUcpMVp

FWIW, this also happens on wordpress.com sites and in fact is where the screencast was taken.  Same results in Jetpack as well though. (both in safari)
